docs/how-to/how-to-run-rootless-vmm.md

```diff
@@ -0,0 +1,28 @@
+## Introduction
+To improve the security, Kata Container runtime supports running the VMM process (currently only QEMU) as a non-root user.
+This doc describes how to enable the rootless VMM mode and its limitations.
+
+## Pre-requisites
+The permission and ownership of the `kvm` device node (`/dev/kvm`) need to be configured to:
+```
+crw-rw---- 1 root kvm
+```
+use the following commands:
+```
+sudo groupadd kvm -r
+sudo chgrp kvm /dev/kvm
+sudo chmod 660 /dev/kvm
+```
+
+## Configure rootless VMM
+By default, the VMM process still runs as root user. There are two ways to enable rootless VMM:
+1. Set the `rootless` flag to `true` in the `configuration.toml`.
+2. Set the Kubernetes annotation `io.katacontainers.hypervisor.rootless` to `true`.
```

Comment:
Please see my comments here : https://github.com/kata-containers/kata-containers/pull/2432/commits/d2dbdaba5fd992bc4a43e4fcd06bf887d7cc3a67#r701717117

Reply:
@amshinde Sorry I couldn't find any comment following the link above. Could you please verify the link or put the comment here?

Reply:
@amshinde, correct me if I'm mistaken, but your comment was related to using the `rootless_hypervisor` instead of only using `rootless`, as the name may be misleading.

I  agree with @amshinde on this topic and I'd appreciate if this could be updated.

Reply:
@fidencio just to confirm, are you suggesting to change the flag from `rootless` to `rootless_hypervisor`?

Reply:
`io.katacontainers.hypervisor.rootless_hypervisor` uhmmm  :thinking:   `hypervisor` word twice ?  weird...

Reply:
Yeah, you convinced me, @devimc. :-)

Reply:
@fengwang666 with the setting in the `configuration.toml`, I get rootless containers regardless of the annotation you refer to here. Are you sure it's necessary?